wordle-web/scripts/hurdle.ts

```diff
@@ -0,0 +1,9 @@
+export function hurdleInput(input: string): string {
+  if (input === 'n') {
+    return 'Good choice, young hurdler...';
+  } else if (input === 'y') {
+    return 'Haha you fell over!';
+  } else {
+    return 'Invalid input. Please try again.';
+  }
+}

```

Comment:
```suggestion
export function hurdleInput(input: string): string {
  switch (input) {
    case "n":
      return "You cleared the hurdle!";
    case "y":
      return "Haha you fell over!";
    default:
      return "Invalid input. Please try again.";
  }
}
```
You could use a switch statement here instead but not too bad since its only three different choices

Reply:
Thank you! I'll make the switch in order for easier changes if there are any in the future.